### PR TITLE
ipatests: bump prci templates

### PR DESCRIPTION
**Pull request by netoarmando**
Remove all freeipa-* packages from template:
https://github.com/freeipa/freeipa-pr-ci/commit/bdd98c3b9dba2ce563535d0c91dad38b532441e8

Signed-off-by: Armando Neto <abiagion@redhat.com>

### CODE REVIEW

File: ipatests/prci_definitions/nightly_latest.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
         git_refspec: '{git_refspec}'
         template: &ci-master-latest
           name: freeipa/ci-master-f32
-          version: 0.0.4
+          version: 0.0.5
```
Comment (fcami) [inline ipatests/prci_definitions/nightly_latest.yaml:54]: hi @netoarmando I expect some of the tests in the nightlies to fail with this change.

**Comment by fcami** [comment]
hi @netoarmando thanks for the PR.
Could you please change the symlink from gating to nightly_latest.yaml? I think at least one test case should be adapted for the missing packages.

**Comment by fcami** [comment]
I just did it: https://github.com/freeipa/freeipa/pull/4862

**Comment by fcami** [comment]
ACK based on CI output

**Comment by netoarmando** [comment]
master:

* cc624fb178291062e30a11b8dc6a6b67cbe5dea8 ipatests: bump prci templates

**Pull request closed without merging**